src/classes/AFFL_MultiRecordType_TDTM.cls

```diff
@@ -74,8 +74,18 @@ public class AFFL_MultiRecordType_TDTM extends TDTM_Runnable {
 				for (SObject so : newlist) {
 					Affiliation__c affl = (Affiliation__c)so;
 
-					String lookupFieldLabel = afflMapper.mapAccRecTypeToContactLabel.get(affl.Affiliation_Type__c);
-					String lookupFieldName = afflMapper.contactLabelNames.get(lookupFieldLabel);
+					String lookupLabelOrAPIName = afflMapper.accRecTypeToContactLabel.get(affl.Affiliation_Type__c);
+					String lookupFieldName = afflMapper.contactLabelNames.get(lookupLabelOrAPIName);
+          
+					// If lookupFieldName of the Contact is not found, check if lookupLabelOrAPIName is an API, if yes assign it to lookupFieldName
+					// such that the API name can be used to populate primay Affiliation lookup is populated on contact even when language is not English
+						
+      if ((lookupLabelOrAPIName != NULL && lookupLabelOrAPIName != '') && (lookupFieldName == '' || lookupFieldName == NULL)) {
```

Comment:
The padding still looks off to me...

![Screen Shot 2019-11-01 at 4 09 23 PM](https://user-images.githubusercontent.com/1894238/68056846-22403900-fcc2-11e9-904c-1f2030ce3f49.png)

Reply:
Hmm when I pull down I see it the right indentation, I tried to ix it again. Can you please give it another shot? Thanks.